server/integration_tests/test_data/e2e_edge_cases2/californiacountieswiththebiggestincreaseinincomelevels/chart_config.json

```diff
@@ -10,36 +10,3642 @@
                 "tiles": [
                   {
                     "comparisonPlaces": [
+                      "geoId/06037",
+                      "geoId/06073",
+                      "geoId/06065",
+                      "geoId/06059",
                       "geoId/06085",
+                      "geoId/06001",
+                      "geoId/06071",
+                      "geoId/06067",
+                      "geoId/06013",
+                      "geoId/06077"
+                    ],
+                    "statVarKey": [
+                      "Count_Person_15OrMoreYears_WithIncome10"
```

Comment:
This feels like a loss. Currently we have "Household Median Income", which is reasonable.

Reply:
This is fixed now